lib/users/utils.js

```diff
@@ -263,7 +262,7 @@ function isNullOrUndefined(value) {
 }
 
 exports.readJSON = readJSON;
-exports.JSONToStr = JSONToStr;
+exports.jsonToStr = jsonToStr;
```

Comment:
why? now `writeJSON` would want to be `writeJson` for consistency? One just needs to pick a convention for acronyms and go with it

Reply:
Fails lint test for camelCase. It `writeJSON` is consistent with that

Reply:
that's a silly test though. consistency of `JSON` should take precedence over the rule that function names should start lowercase

Reply:
Sure

Reply:
Why don't we just kill the linter then? I don't see a point in abiding by linting rules if we just decide something is silly

Reply:
nothing works 100%. linters are a net positive when ignored appropriately

Reply:
> nothing works 100%. linters are a net positive when ignored appropriately

Disagree, this is a pretty clear example of following the linter. Everything about it is personal preference which shouldn't play a role in linting unless you write the linter to account for that (which we didn't here).